tasks/gulp/compile-scss.js

```diff
@@ -22,7 +22,7 @@ gulp.task('scss:compile-all-govuk-and-hmrc', () => gulp.src(`${configPaths.src}a
   .pipe(plumber(errorHandler))
   .pipe(sourcemaps.init())
   .pipe(sass({
-    includePaths: ['node_modules'],
+    includePaths: ['node_modules', 'node_modules/govuk-frontend/dist', 'node_modules/govuk-frontend/dist/govuk/components'],
```

Comment:
we have two types of relative imports at the moment `../../govuk-frontend` and `../../../../govuk-frontend` which - I think govuk-frontend gets around issues with load paths by not importing any css from node_modules in their css - they just have their own css / stuff vendored in that's a single file

the way this works, we don't actually want to use any css from either of those new paths, we just want to be able to relatively traverse to places in govuk-frontend from hmrc-frontend css when we're compiling it when the hmrc-frontend css is not inside node_modules - all our consumers compile the hmrc css in the context of hmrc-frontend being inside a folder (like node_modules or lib in the case of webjars) where govuk-frontend will be also in that folder, so relative traversals for them work